src/util/Injected.js

```diff
@@ -406,6 +408,10 @@ exports.LoadUtils = () => {
             mediaData.type = 'document';
         }
 
+        if(mediaInfo.filesize && maxFileSize > mediaInfo.file){
+            throw new Error('Media size exceeds client upload limit');
+        }
```

Comment:
Please return information about the size limit for the media type.
And add a function that give an option to check the size limit for media type

Reply:
Yes!, i am currently testing locally and will push soon

Reply:
Ok i've added a new function, added to index.d.ts , to client.js (getUploadLimits) and updated the error